lightning/src/ln/channelmanager.rs

```diff
@@ -441,6 +445,14 @@ enum BackgroundEvent {
 	ClosingMonitorUpdate((OutPoint, ChannelMonitorUpdate)),
 }
 
+pub(crate) enum MonitorUpdateCompletionAction {
+	/// Indicates that a payment ultimately destined for us was claimed and we should surface an
+	/// [`events::Event::PaymentClaimed`] to the user iff no such event has already been surfaced.
+	PaymentClaimed { payment_hash: PaymentHash },
+	/// Indicates an [`events::Event`] should be surfaced to the user.
+	SurfaceEvent { event: events::Event },
```

Comment:
So regarding:

1. No, I just wasn't sure whether this is kind of redundant, even though they do different things. Still think that it would make it a bit clearer to not name it after the event but follow the `VerbNoun` structure of `SurfaceEvent` for consistency's sake.
2. I'm also fine with emit, as long as we end up with the same terminology for the same thing eventually.